sxt/multiexp/pippenger2/in_memory_partition_table_accessor.h

```diff
@@ -35,6 +35,8 @@ namespace sxt::mtxpp2 {
 //--------------------------------------------------------------------------------------------------
 template <bascrv::element T>
 class in_memory_partition_table_accessor final : public partition_table_accessor<T> {
+  static constexpr unsigned num_entries = 1u << 16u;
```

Comment:
There are a number of instances in the code where the number of entries get set to `1u << 16u`. Do you think adding a constants component in `multiexp/pippenger2` would have any value? I'm not sure if the number of entries supported will ever change or if incorrectly setting the number of entries will be difficult to debug, so it might not be worth it.

Reply:
done